src/cli.ts

```diff
@@ -37,6 +37,12 @@ const optionDefinitions = [
     description:
       'Command to create reference before testing. Default: null (Do not create reference before test).',
   },
+  {
+    name: 'output-profile',
+    type: String,
+    defaultValue: null,
+    description: 'Measure the execution time and output the report as JSON.',
```

Comment:
I want to describe that this option is a path to output file, not boolean, or something.

For example:
```suggestion
    description: 'Path to profiler output file. If present, measure the execution time and output the report as JSON.',
```

Reply:
Oh, I think it was originally implemented as a bool flag and has been retained. I'll fix it.